packages/sdk/tasks/jarvisFix.ts

```diff
@@ -0,0 +1,92 @@
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { task, types } from "hardhat/config";
+
+task("jarvis-fix", "deploy new strategy for jarvis 2brl pool")
+  .addParam("signer", "Named account to use fo tx", "deployer", types.string)
+  .setAction(async (taskArgs, hre) => {
+    let signer: SignerWithAddress;
+    try {
+      signer = await hre.ethers.getNamedSigner(taskArgs.signer);
+    } catch {
+      throw `Invalid 'signer': ${taskArgs.signer}`;
+    }
+
+    // @ts-ignore
+    const fuseModule = await import("../tests/utils/fuseSdk");
+    const sdk = await fuseModule.getOrCreateFuse();
+
+    const jarvisComptroller = "0x31d76A64Bc8BbEffb601fac5884372DEF910F044";
+    const dddAddress = "0x84c97300a190676a19D1E13115629A11f8482Bd1";
+    const epxAddress = "0xaf41054c1487b0e5e2b9250c0332ecbce6ce9d71";
+    const twobrl = "0x1B6E11c5DB9B15DE87714eA9934a6c52371CfEA9";
+    const dddFlywheelAddress = "0x851Cc0037B6923e60dC81Fa79Ac0799cC983492c";
+    const epxFlywheelAddress = "0xC6431455AeE17a08D6409BdFB18c4bc73a4069E4";
+    const dddFlywheel = sdk.createFuseFlywheelCore(dddFlywheelAddress);
+    const epxFlywheel = sdk.createFuseFlywheelCore(epxFlywheelAddress);
+    const marketAddress = await sdk
+      .createComptroller("0x31d76A64Bc8BbEffb601fac5884372DEF910F044", signer)
+      .callStatic.cTokensByUnderlying(twobrl);
+    const cToken = await sdk.createCErc20PluginRewardsDelegate(marketAddress);
+    console.log({ marketAddress });
+
+    // Step 1: Deploy Fresh Strategy with marketAddress=rewardsDestination
+    const strategyArgs = [
+      "0x1B6E11c5DB9B15DE87714eA9934a6c52371CfEA9", //_asset,
+      dddFlywheelAddress, //_dddFlywheel,
+      epxFlywheelAddress, //_epxFlywheel,
+      "0x8189F0afdBf8fE6a9e13c69bA35528ac6abeB1af", //_lpDepositor,
+      marketAddress, //_rewardsDestination,
+      ["0x84c97300a190676a19D1E13115629A11f8482Bd1", "0xaf41054c1487b0e5e2b9250c0332ecbce6ce9d71"], //_rewardTokens
+    ];
+    console.log({ strategyArgs });
+
+    const pluginDeployment = await hre.deployments.deploy("DotDotLpERC4626_" + marketAddress, {
```

Comment:
is this the new convention or just a one-time thing? @carlomazzaferro

Reply:
That's the convention. As we've discussed, each market will have its own plugin address